### PR TITLE
TECH-7663 SOLUTION2: Render failing due to Gdk::PixbufError::CorruptImage (Unsupported marker type 0x64) errors

### PR DESCRIPTION
**Pull request by makemp**
Link:
https://livelinktech.atlassian.net/browse/TECH-7663

Alternative solution:
https://github.com/livelink/web-kiosk/pull/5276
Effect: same as linked photos in PR.
(see screens here: #5276)

While working on PR 5267 I discovered that saving corrupted image into a file using Pixbuf's method `save` fixes the image.

### CODE REVIEW

**Comment by allegromanontroppo** [comment]
(See Kondrad's comment 3 here: https://github.com/livelink/morandi-rb/pull/3)

File: lib/morandi/profiled_pixbuf.rb [hunk 1]
```diff
@@ -30,6 +30,15 @@ def initialize(*args)
     end
 
     super(*args)
+  rescue Gdk::PixbufError::CorruptImage => e
+    if args[0].is_a? String
+      temp_path =  Tempfile.new.path
+      pixbuf.save(temp_path, 'jpeg')
```
Comment (makemp) [inline lib/morandi/profiled_pixbuf.rb]: ups :)

Comment (allegromanontroppo) [reply lib/morandi/profiled_pixbuf.rb]: what's the oops? That you've got a variable reference to `pixbuf`?

Comment (makemp) [reply lib/morandi/profiled_pixbuf.rb]: Now I have, yes :)

File: lib/morandi/profiled_pixbuf.rb [hunk 2]
```diff
@@ -12,6 +12,15 @@ def valid_jpeg?(filename)
     false
   end
 
+  def self.from_string(string, loader: nil, chunk_size: 4096)
```
Comment (makemp) [inline lib/morandi/profiled_pixbuf.rb]: From WK `lib/core_ext/gdk/pixbuf.rb`

File: lib/morandi/profiled_pixbuf.rb (hunk 1 above)
Comment (allegromanontroppo) [reply lib/morandi/profiled_pixbuf.rb]: Will the TempFile get cleaned up, do you think?

**Comment by allegromanontroppo** [comment]
Maybe don't worry about bumping the version, because we've not yet published my change from yesterday. But please update the changelog.

File: lib/morandi/profiled_pixbuf.rb (hunk 1 above)
Comment (makemp) [reply lib/morandi/profiled_pixbuf.rb]: It will be cleaned but i think we should explicity close it. As good practice says. Thx

**Comment by makemp** [comment]
@allegromanontroppo Changelog updated. I could write a test for this behaviour if you wish.

**Comment by allegromanontroppo** [comment]
Sorry that your naked selfie didn't come out correctly.

**Pull request merged**